cli/tokens.go

```diff
@@ -46,7 +46,7 @@ func (r *RootCmd) tokens() *serpent.Command {
 
 func (r *RootCmd) createToken() *serpent.Command {
 	var (
-		tokenLifetime time.Duration
+		tokenLifetime string
```

Comment:
@defelmnq @johnstcn WDYT about defining a new type in [coder/serpent](https://github.com/coder/serpent), `time.LongDuration`?

Reply:
Could be done as a follow-up!